### PR TITLE
BLD: run pytest on wheels and sdist

### PR DESCRIPTION
**Pull request by yut23**
## PR Summary

The changes to pyproject.toml are needed to allow running the tests on the installed version of the code. The `--ignore` paths are resolved relative to the current directory before being compared to the test files if they're not already absolute. This works fine when running from the repository root, but pytest would have to be run from `site-packages/` to get the same relative file paths and it would try to run the tests of every installed package.

`--rootdir .` makes pytest collect the tests from the installed package instead of the parent directory of the configuration file.

## PR Checklist

- [ ] New features are documented, with docstrings and narrative docs
- [ ] Adds a test for any bugs fixed. Adds tests for new features.

### CODE REVIEW

**Review by neutrinoceros — APPROVED** [review]
Thank you for continuing this thread ! Obviously jobs are longer now: builds complete in about 45min (wall time) instead of 15, but that's a perfectly reasonable amount of time to wait for a release IMO. I'm approving now, but I'll give some time (say a week or two) for others to review.

**Review by chrishavlin — APPROVED** [review]
thanks for your work on this, @yut23 !

**Pull request merged**
